frontend/.changeset/tasty-lies-shop.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+Updated title and ERD view header title
```

Comment:
It might be better if one of the titles is clarified as an HTML element. What do you think?

```suggestion
Update HTML and view header titles in ERD
```

Reply:
Oh, that sounds much better! Thanks a lot!
fixed: https://github.com/liam-hq/liam/pull/242/commits/079e03ca4d35f3fc64b59fda034f1b4d0a4136ee